WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift

```diff
@@ -231,12 +231,20 @@ extension SiteTagsViewController {
 
     private func delete(_ tag: PostTag) {
         let tagsService = PostTagService(managedObjectContext: ContextManager.sharedInstance().mainContext)
-        tagsService.delete(tag, for: blog)
+        refreshControl?.beginRefreshing()
+        tagsService.delete(tag, for: blog, success: {
+            self.refreshControl?.endRefreshing()
+            self.tableView.reloadData()
+        }, failure: nil)
```

Comment:
If the call fails would we still want to call `self.refreshControl?.endRefreshing()`?